### ISSUE TITLE
Typescript: ResourceActionOptions lists method option as required

### ISSUE THREAD

**Issue by dboune** [christianmalek/vuex-rest-api#39]
- Are you using the newest version of vuex-rest-api? `YES: 2.2.1`
- Did you read the README? `YES`
- Did you check the existing issues? Maybe there's already a solution to your problem. `YES`

In this case the `method` property is set by the `get` method of Vapi, so method is not required. Additionally `Resource.add()` defaults the method property to `get`.

```js
items.get({
  action: 'listItems',
  property: 'items',
  path: '/'
})
```

> TS2345: Argument of type '{ action: string; property: string; path: string; }' is not assignable to parameter of type 'ResourceActionOptions'.
  Property 'method' is missing in type '{ action: string; property: string; path: string; }'.

**Comment by christianmalek** [comment]
Hello @dboune,

I'm sorry for the long response time. I fixed the bug in 606871517153ca1f4b2c2cdbcb5b83618bdccdee.

> In this case the method property is set by the get method of Vapi, so method is not required. Additionally Resource.add() defaults the method property to get.
I don't want to change this because it could break the application from someone who depends on this default value.

**Comment by dboune** [comment]
No worries, and thank you!
